packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -115,7 +115,15 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
     private async _create(id: CoreId, content: Request, peer: CoreAddress) {
         const canCreateResult = await this.canCreate({ content, peer });
 
-        if (canCreateResult.isError()) throw canCreateResult.error;
+        if (canCreateResult.isError()) {
+            if (canCreateResult.error.code === "error.consumption.validation.inheritedFromItem") {
+                throw ConsumptionCoreErrors.requests.inheritedFromItem(
+                    "Some child items have errors. If this error occurred during the specification of a Request, call 'canCreate' to get more information."
```

Comment:
we actually know that this occurred during _create, so we can simplify this

```suggestion
                    "Some child items have errors. Call 'canCreate' to get more information."
```